studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -156,12 +156,30 @@ export const genDefaultQuery = (table: LogsTableName, filters: Filters) => {
     default:
       return `select id, ${table}.timestamp, event_message from ${table}
   ${where}
-  limit 100          
+  limit 100
   `
   }
 }
 
 /**
  * SQL query to retrieve only one log
  */
-export const genSingleLogQuery = (table: LogsTableName, id: string) => `select id, timestamp, event_message, metadata from ${table} where id = '${id}' limit 1`
\ No newline at end of file
+export const genSingleLogQuery = (table: LogsTableName, id: string) =>
+  `select id, timestamp, event_message, metadata from ${table} where id = '${id}' limit 1`
+
+/**
+ * Determine if we should show the user an updgrade prompt while browsing logs
+ *
+ * There's only 1440 minutes in a day, but when we select "Last day" from the dropdown,
+ * it will set that to mean yesterday at midnight.
+ * So the max possible time from that midnight to the upcoming midnight is 2880
+ */
+export const maybeShowUpgradePrompt = (from: any) => {
```

Comment:
@Ziinc added a helper function here as you suggested!

Ts help on that any is much appreciated!